### PR TITLE
[DO-2554]Add information about PEP types in DO FAQ

### PR DESCRIPTION
**Pull request by rpersten**

### CODE REVIEW

File: src/components/pages/help/faqs.jsx [hunk 1]
```diff
@@ -116,6 +116,47 @@ const faqs = [
       </Translation>
     ),
   },
+  {
+    id: 4,
```
Comment (roman-tiukh) [inline src/components/pages/help/faqs.jsx:120]: id should be unique in list

File: src/components/pages/help/faqs.jsx [hunk 2]
```diff
@@ -116,6 +116,47 @@ const faqs = [
...
+                1. {t('faq.answer.q4ForeignPolitically')}
+              </div>
+              <div className="pl-10">
+                1. {t('faq.answer.q4PoliticallyExposedPersons')}
+              </div>
+              <div className="pl-10">
+                1. {t('faq.answer.q4AssociatedPersonsOf')}
+              </div>
+              <div className="pl-10">
+                1. {t('faq.answer.q4ClosePersonsOf')}
```
Comment (roman-tiukh) [inline src/components/pages/help/faqs.jsx:151]: all items have `1` number?

File: src/components/pages/help/faqs.jsx [hunk 3]
```diff
@@ -116,6 +116,47 @@ const faqs = [
...
+              </div>
+            </div>
+
+          </>
```
Comment (roman-tiukh) [inline src/components/pages/help/faqs.jsx:155]:
```suggestion
            </div>
          </>
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/components/pages/help/faqs.jsx (hunk 3 above)
Comment (rpersten) [reply src/components/pages/help/faqs.jsx:155]: Done!

File: src/components/pages/help/faqs.jsx (hunk 2 above)
Comment (rpersten) [reply src/components/pages/help/faqs.jsx]: Done!

File: src/locales/en/translation.json [hunk 4]
```diff
@@ -147,13 +147,21 @@
...
       "q3InfoContainedInTheRegister": "The information contained in the Register is based on evidence collected by the Anti-Corruption Action Centre from publicly available reliable",
       "q3TheUsageOfTheRegisterRecommended": "The usage of the Register is recommended by the National Bank of Ukraine and the State Financial Monitoring Service of Ukraine as one of the information sources for the primary financial monitoring subject in the national public figures identification among their clients.",
-      "q4DOIsATechnicalPartner": "Data Ocean is a technical partner of the Anti-Corruption Action Centre.\nData Ocean provides a technical support for the registry and both resources; it has the exclusive right to sell access to the PEP register for commercial organizations.",
+      "q4DOIsATechnicalPartner": "Data Ocsean is a technical partner of the Anti-Corruption Action Centre.\nData Ocean provides a technical support for the registry and both resources; it has the exclusive right to sell access to the PEP register for commercial organizations.",
```
Comment (roman-tiukh) [inline src/locales/en/translation.json:164]:
```suggestion
      "q4DOIsATechnicalPartner": "Data Ocean is a technical partner of the Anti-Corruption Action Centre.\nData Ocean provides a technical support for the registry and both resources; it has the exclusive right to sell access to the PEP register for commercial organizations.",
```

File: src/components/pages/help/faqs.jsx (hunk 3 above)
Comment (roman-tiukh) [reply src/components/pages/help/faqs.jsx:155]: I don't see this change

Comment (rpersten) [reply src/components/pages/help/faqs.jsx:155]: Changed

File: src/locales/en/translation.json (hunk 4 above)
Comment (rpersten) [reply src/locales/en/translation.json:164]: Changed

**Pull request closed without merging**